src/rendezvous_mediator.rs

```diff
@@ -92,7 +85,11 @@ impl RendezvousMediator {
                 for host in servers.clone() {
                     let server = server.clone();
                     futs.push(tokio::spawn(async move {
-                        if let Err(err) = Self::start(server, host).await {
```

Comment:
Let's use another loop. Let's define a new static variable in config.rs.

```
pub const NATS_SERVERS: &str = match option_env!("NATS_SERVER") {
    Some(server) if !server.is_empty() => [server],
    _ => ["nats://nats.rustdesk.com"],
};
```

 And should we connect to multiple nats servers the same time? We should loop to connect to only one avaiable nats server.

Reply:
We have to fork then.

Reply:
OK. Let me check if I understand it right. This means we should consider the NATS server/cluster as a separate service, not just an extra port of a rendezvous server. Thus connecting to any of them would be enough.

Reply:
Yes, it is a seperate service, just like https://azure.microsoft.com/en-us/products/iot-hub

Reply:
I got it. 

BTW, would it be acceptable if we send `NATS_SERVERS` in `RegisterPkResponse`? Since we could connect to MQ only after we get `mq_token`, IMO it is reasonable to also get `mq_servers` in the same way.